rest-service/manager_rest/rest/resources_v1/executions.py

```diff
@@ -201,6 +201,58 @@ def post(self, **kwargs):
                 execution.id = execution_id
             sm.put(execution)
             messages = []
+            if (
+                force_status in ExecutionState.STATES
+                and force_status not in ExecutionState.WAITING_STATES
+                and force_status != ExecutionState.PENDING
+            ):
+                if deployment_id:
+                    deployment = sm.get(models.Deployment, deployment_id)
```

Comment:
`deployment` is already retrieved here (see 15 lines above), it'd be good if we were able to avoid additional queries